docs/pages/access-controls/teleport-policy/policy-how-to-use.mdx

```diff
@@ -3,16 +3,16 @@ title: Teleport Policy Usage
 description: Using Access Graph with Teleport Policy.
 ---
 
-Policy use with the Access Graph provides a framework for visualizing and managing 
+The Teleport Policy feature provides a framework for visualizing and managing 
```

Comment:
Is Teleport Policy a feature or a product? We seem to refer to it as both, but then we also say that access graph is a feature. That would make access graph a feature of a feature. Kind of confusing, no?

Reply:
Thank you Zac. The term 'Teleport Policy feature' as well as using 'Access Graph abbreviation in repeated use on the same page was requested by the Marketing folks. I've added you to the [Google Doc](https://docs.google.com/document/d/1EazxSDyMFQ4hY2E7M7x4Vw04vEI4a0XTddKsKcEp-lc/) ( I don't think Eddie is in GitHub).

Reply:
The doc says to use _Teleport Policy’s Access Graph feature_, which I interpret to mean that access graph is a **feature** of the Teleport Policy **product**.

Reply:
How about: _Teleport Policy provides a framework for visualizing and managing access controls across an organization’s infrastructure._